docs/test.ts

```diff
@@ -78,6 +78,10 @@ tracer.init({
     { sampleRate: 0.5, service: 'ba?', name: 'ba?.*', maxPerSecond: 10 }
   ],
   service: 'test',
+  serviceMapping: {
+    a: 'aa',
+    b: 'bb'
```

Comment:
I think it'd increase readability for us if we put better examples here, like we do in the docs, or at least something like: `plugin_name:service_name`

Reply:
I just copied the form that the original tests had. I could make it something more verbose though, if that's what we prefer. 🤷🏻